spidermon/contrib/validation/translator.py

```diff
@@ -12,6 +21,14 @@ def translate_messages(self, messages):
         return [self.translate_message(m) for m in messages]
 
     def translate_message(self, message):
+        if (
+            schematics_installed
+            and message
+            and type(message) is schematics.datastructures.FrozenList
+        ):
+            # Necessary to handle model-level custom validators correctly
+            message = message[0].summary
```

Comment:
It would be nice if we can move schematic related logic in it's own file & class.